src/controllers/bus.controller.js

```diff
@@ -0,0 +1,142 @@
+import { paginate } from 'paginate-info';
+import model from '../database/models';
+
+const createBus = async (req, res) => {
+  try {
+    const existingBus = await model.Bus.findOne({
+      where: { busPlate: req.body.busPlate },
+    });
+    if (existingBus) {
+      return res.status(400).json({
+        status: 400,
+        message: res.__('Bus already exists in the system.'),
+      });
+    }
+    const bus = await model.Bus.create(req.body);
```

Comment:
may you just extract the Bus table form the model outside of the call
so that you don't have to always write `model.Bus` every time you want to access the table